bluesky/traffic/route.py

```diff
@@ -928,6 +878,7 @@ def newcalcfp(self):
 
         # Find longest segment without altitude constraints
 
+        desslope = clbslope = 1.

```

Comment:
code analyzer complains about possible non-initialization of `desslope` and `clbslope`